### PR TITLE
DOCSP-29606 minimum version

### PR DESCRIPTION
**Pull request by davemungo**
Staging
[limitations](https://docs-mongodbcom-staging.corp.mongodb.com/cluster-sync/docsworker-xlarge/DOCSP-29606-minimum-version-v1.0/reference/limitations/#general-limitations)
[release notes 1.3](https://docs-mongodbcom-staging.corp.mongodb.com/cluster-sync/docsworker-xlarge/DOCSP-29606-minimum-version-v1.0/release-notes/1.3/)
[release notes 1.2](https://docs-mongodbcom-staging.corp.mongodb.com/cluster-sync/docsworker-xlarge/DOCSP-29606-minimum-version-v1.0/release-notes/1.2/)

[jira](https://jira.mongodb.org/browse/DOCSP-29606)

[build](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=645121f27efd2382b3381275)

### CODE REVIEW

**Comment by davemungo** [comment]
@alan-zheng-mdb  (From Slack)
After the announcement of support for earlier versions in 1.2, the minimum version is potentially confusing. The message about earlier migrations is also in danger of getting lost in past release notes now that we are on 1.3.  When I bumped the minimum version on the limitations page also added the text from the 1.2 announcement. If that's a bad idea, let me know and I'll remove the note from the Limitations page

File: source/includes/fact-minimum-versions.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Starting in version 1.2, ``mongosync`` can sync between clusters running 
+older (pre-6.0) releases of MongoDB Server.  Use of this feature
```
Comment (jmd-mongo) [inline source/includes/fact-minimum-versions.rst:2]: [nit]

Consider either "Using this feature..." or simply, "This feature requires...".

**Review by jmd-mongo — APPROVED** [review]
Thanks for the updates, @davemungo! This one LGTM mod a nit.

Thanks!
Joe

File: source/includes/fact-minimum-versions.rst (hunk 1 above)
Comment (davemungo) [reply source/includes/fact-minimum-versions.rst]: Done

**Review by alan-zheng-mdb — APPROVED** [review]
LGTM

**Pull request merged**